miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/MiniAppWebChromeClient.kt

```diff
@@ -21,8 +31,41 @@ internal class MiniAppWebChromeClient(context: Context) : WebChromeClient() {
         super.onReceivedTitle(webView, title)
     }
 
+    // region geolocation
```

Comment:
What is the intent/purpose of `//region geolocation`?

Reply:
it's just a coding style specifies the code block for this feature purpose. By specifying the code block, we can collapse the code visibility in AS.

Reply:
Okay. 
I could also suggest using the logical purpose-based grouping as a sequence for methods in the class as recommended as well in the kotlin docs.

Reply:
Oh, can you refer me to that kotlin doc?

Reply:
https://developer.android.com/kotlin/style-guide#top-level_declarations